### PR TITLE
feat: Add columns to track job execution time

### PR DESCRIPTION
**Pull request by johnjcsmith**
Adds two new columns on `job` for latency metrics:

- `resulted_at` the time that a job result was persisted.
- `function_execution_time` the time in milliseconds of a function handler execution.

Will follow up with #29 to update `ts-client` to record the runtime handler latency.

### CODE REVIEW

File: control-plane/drizzle/0020_talented_pandemic.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE "jobs" ADD COLUMN "runtime_latency_ms" integer;
```
Comment (johnjcsmith) [inline control-plane/drizzle/0020_talented_pandemic.sql:1]: I am curious of _"runtime"_ latency is the correct naming here?
Was also thinking _"handler"_ latency might make sense. 

image

Comment (johnjcsmith) [reply control-plane/drizzle/0020_talented_pandemic.sql:1]: Latency probably isn't the correct term at all here.

Comment (johnjcsmith) [reply control-plane/drizzle/0020_talented_pandemic.sql]: Renaming this to `function_execution_time`

File: control-plane/drizzle/0019_early_silver_sable.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DROP TABLE "users";--> statement-breakpoint
```
Comment (johnjcsmith) [inline control-plane/drizzle/0019_early_silver_sable.sql:1]: Assuming this just wasn't committed previously?

Comment (nadeesha) [reply control-plane/drizzle/0019_early_silver_sable.sql:1]: Yes. Probably. Thanks!

File: control-plane/drizzle/0019_early_silver_sable.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+ALTER TABLE "jobs" ADD COLUMN "resulted_at" timestamp with time zone;--> statement-breakpoint
+ALTER TABLE "jobs" ADD COLUMN "function_execution_time_ms" integer;--> statement-breakpoint
+ALTER TABLE "jobs" ADD COLUMN "service" varchar(1024);
```
Comment (nadeesha) [inline control-plane/drizzle/0019_early_silver_sable.sql:4]:
```suggestion
ALTER TABLE "jobs" ADD COLUMN IF NOT EXISTS "service" varchar(1024);
```

This was created, but not committed as a part of a previous deployment. This should fix your build.

Comment (johnjcsmith) [reply control-plane/drizzle/0019_early_silver_sable.sql]: Nice, thanks!

**Review by nadeesha — APPROVED** [review]

**Pull request merged**